assemblies/psw-ce/src/main/resources/log4j2.xml

```diff
@@ -35,6 +34,7 @@
         <!-- Append messages to the console -->
         <!-- ============================== -->
         <Console name="CONSOLE">
+            <ThresholdFilter level="INFO" onMatch="ACCEPT" onMismatch="DENY"/>
```

Comment:
I don't think you need to specify any attributes... the default for onMatch is NEUTRAL.